examples/hikari_lightbulb_basic_queue/main.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+import hikari
+import lightbulb
+import lavasnek_rs
+
+PREFIX = "," # Should probably set this to an environment variable
+TOKEN = os.environ["DISCORD_TOKEN"]
+
+
+class Data:
+    """Global data shared across the entire bot, used to store dashboard values."""
+    def __init__(self) -> None:
+        self.lavalink: lavasnek_rs.Lavalink = None
+
+
+class MusicBot(lightbulb.Bot):
+    """Subclass of lightbulb's Bot object, used to store the lavalink client."""
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.data = Data()
+
+# You may want to enable ALL intents here
+bot = MusicBot(token=TOKEN, prefix=PREFIX)
```

Comment:
Importing the constants from this file will re-run it, you should put this behind `if __name__ == "__main__":` with an else, redefining `bot` with `banner=None`

Reply:
Resolved by using a separate file for constants